NorthstarDLL/server/serverpresence.cpp

```diff
@@ -176,6 +176,12 @@ void ServerPresenceManager::SetAuthPort(const int iAuthPort)
 	m_ServerPresence.m_iAuthPort = iAuthPort;
 }
 
+void ServerPresenceManager::SetId(const std::string sServerId)
+{
+	// update id
+	m_ServerPresence.m_sServerId = sServerId;
+}
+
```

Comment:
why?

Reply:
Addressed in 603dfc72a109d2043e456271abf3f33d259eb2c0